openff/toolkit/topology/molecule.py

```diff
@@ -2792,6 +2793,62 @@ def generate_conformers(
                 )
             )
 
+    def apply_elf_conformer_selection(
+        self,
+        percentage: float = 2.0,
+        limit: int = 10,
+        toolkit_registry: Optional[
+            Union[ToolkitRegistry, ToolkitWrapper]
+        ] = GLOBAL_TOOLKIT_REGISTRY,
+        **kwargs,
+    ):
+        """Applies the `ELF method<https://docs.eyesopen.com/toolkits/python/quacpactk/
+        molchargetheory.html#elf-conformer-selection>`_ to select a set of diverse
+        conformers which have minimal electrostatically strongly interacting functional
+        groups from a molecules conformers.
+
+        Notes
+        -----
+        * The input molecule should have a large set of conformers already
+          generated to select the ELF conformers from.
+        * The selected conformers will be retained in the `conformers` list
+          while unselected conformers will be discarded.
+
+        See Also
+        --------
+        ``OpenEyeToolkitWrapper.apply_elf_conformer_selection``
+        ``RDKitToolkitWrapper.apply_elf_conformer_selection``
```

Comment:
This makes docs fail to build. Not sure why. Yay rst. 
```suggestion
        OpenEyeToolkitWrapper.apply_elf_conformer_selection
        RDKitToolkitWrapper.apply_elf_conformer_selection
```